### ISSUE TITLE
detectEdges function from StructuredEdgeDetection class (contrib module) cause crash in app

### ISSUE THREAD

**Issue by satheesh-agnext** [quickbirdstudios/opencv-android#52]
I'm using the Gradle dependency "implementation 'com.quickbirdstudios:opencv:4.3.0-contrib'" to use the StructuredEdgeDetection in the android app which is from contrib module. It was working perfectly when I'm using " implementation 'com.quickbirdstudios:opencv:3.4.4-contrib'". But I need to use opencv 4.3.0. Please help

**Comment by nasirky** [comment]
Thank you for reaching out. Can you please share the crash log?

**Comment by satheesh-agnext** [comment]
Only log I'm getting is Crash log : A/libc: Fatal signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x71ea04cffc in tid 15939 (Thread-3), pid 15816. 
After that app process is killed and not getting any other info. Are the .so files are proper with contrib module ?

**Comment by nasirky** [comment]
@satheesh-agnext Yes the source files should be fine. We will be releasing a new version of opencv (4.5.2) until end of the next week. I would recommend you try with that and if the crash exists, then we can see what can be done.

**Comment by nasirky** [comment]
We have just released 4.5.3 via MavenCentral. Please go through the README file on how to integrate it. In case the issue still persists, please let us know.
